SigmaBank/Sources/Clients/Protocols/PayeeClient.swift

```diff
@@ -0,0 +1,22 @@
+class GetPayeesbyPhoneNumberResult {
+    
+    let payees: [Payee]
+    
+    let pageToken: String
+
+    init(payees: [Payee], pageToken: String) {
+        self.payees = payees
+        self.pageToken = pageToken
+    }
+
+}
+
+protocol PayeeClientProtocol {
+
+    func getPayeesByPhoneNumber(phoneNumberSubstring: String, pagination: PaginationModel, token: AuthenticationToken) async throws -> GetPayeesbyPhoneNumberResult
+    
+    func getPayeeByCardNumber(cardNumber: CardNumber, token: AuthenticationToken) async throws -> Payee
```

Comment:
Если руководствоваться гайдами Apple, то нейминг должен читаться, как предложения

```suggestion
    func getPayeeBy(cardNumber: CardNumber, token: AuthenticationToken) async throws -> Payee
```